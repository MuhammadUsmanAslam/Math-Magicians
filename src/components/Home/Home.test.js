import renderer from "react-test-renderer";
import Home from "./Home";

test("renders Home component", () => {
	const tree = renderer.create(<Home></Home>).toJSON();

	expect(tree).toMatchSnapshot();
});
