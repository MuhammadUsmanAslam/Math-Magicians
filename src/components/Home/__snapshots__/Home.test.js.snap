// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home component 1`] = `
<div
  className="home"
>
  <h1
    className="home__heading"
  >
    Welcome to our page!
  </h1>
  <p
    className="home__para"
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quasi, distinctio. Dolorem unde distinctio reprehenderit ducimus blanditiis magnam deserunt, soluta, voluptatibus tenetur saepe deleniti aut. Aut odit cupiditate ab aliquam non.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quasi, distinctio. Dolorem unde distinctio reprehenderit ducimus blanditiis magnam deserunt, soluta, voluptatibus tenetur saepe deleniti aut. Aut odit cupiditate ab aliquam non.
  </p>
  <p
    className="home__para"
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quasi, distinctio. Dolorem unde distinctio reprehenderit ducimus blanditiis magnam deserunt, soluta, voluptatibus tenetur saepe deleniti aut. Aut odit cupiditate ab aliquam non.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quasi, distinctio. Dolorem unde distinctio reprehenderit ducimus blanditiis magnam deserunt, soluta, voluptatibus tenetur saepe deleniti aut. Aut odit cupiditate ab aliquam non.
  </p>
</div>
`;
